app/controllers/dams_resource_controller.rb

```diff
@@ -40,6 +40,10 @@ def show
     # generate facet collection list for collection page only
     models = @document["active_fedora_model_ssi"]
     if models.include?("DamsAssembledCollection") || models.include?("DamsProvenanceCollection") || models.include?("DamsProvenanceCollectionPart") 
+      other_rights_query = "(otherRights_tesim:localDisplay OR otherRights_tesim:metadataDisplay) AND collections_tesim:#{params[:id]}"
+      @res, @doc = get_search_results(q: other_rights_query, rows: 1)
+      @metadata_only = @res.response['numFound'].zero? ? false : true
```

Comment:
Doesn't `.zero?` return true or false already? Do we need the ternary part? (` ? false : true `)

Reply:
Oh i see what you're doing here, you actually want the inverse. nevermind :) I suppose an alternative to the ternary is `@metadata_only = !@res.response['numFound'].zero?` but that's pretty subjective as far as what's more readable..

Reply:
@mcritchlow I think your alternative is better.  I'll update it to use it.